lib/Donors/DonorsList.js

```diff
@@ -18,22 +19,27 @@ export const DonorsList = ({
   formatter: formatterProp,
   id,
   visibleColumns,
+  ...rest
 }) => {
   const stripes = useStripes();
   const canViewOrganizations = stripes.hasPerm('ui-organizations.view');
   const formatter = useMemo(() => {
     return formatterProp || getDonorsListFormatter({ canViewOrganizations });
   }, [canViewOrganizations, formatterProp]);
 
+  const anchoredRowFormatter = (rowProp) => acqRowFormatter(rowProp);
```

Comment:
Why create a function that doesn't do anything when you can pass in the original?

Reply:
updated